### PR TITLE
Support `SHOW COLUMNS FROM mytable`

### PR DESCRIPTION
**Pull request by quodlibetor**

### CODE REVIEW

File: src/parser.rs [hunk 1]
```diff
@@ -1663,6 +1664,29 @@ impl Parser {
...
+    fn parse_show_columns(&mut self) -> Result<Statement, ParserError> {
+        if !self.parse_keyword("FROM") {
+            return self.expected("FROM", self.peek_token());
+        }
+        let table_or_schema = self.parse_object_name()?;
+
+        if let Some(tok) = self.peek_token() {
+            // TODO: show columns from .. from .. is unsupported
+            return self.expected("Nothing", tok.into());
+        }
```
Comment (benesch) [inline src/parser.rs]: tiny optional nit: I think the convention elsewhere is to not include special handling for extraneous stuff and let the syntax error occur on the semicolon.

Comment (quodlibetor) [reply src/parser.rs]: Oh does the parser auto-error if it doesn't consume everything? Okay.

Comment (benesch) [reply src/parser.rs]: Yep, exactly.

**Comment by benesch** [comment]
Merge away!

**Pull request merged**